internal/runners/manifest/version.go

```diff
@@ -0,0 +1,42 @@
+package manifest
+
+import (
+	"github.com/ActiveState/cli/internal/locale"
+	"github.com/ActiveState/cli/pkg/platform/api/buildplanner/model"
+	platformModel "github.com/ActiveState/cli/pkg/platform/model"
+	"github.com/ActiveState/cli/pkg/platform/runtime/artifact"
+)
+
+type resolvedVersion struct {
+	Requested string `json:"requested"`
+	Resolved  string `json:"resolved,omitempty"`
```

Comment:
Currently this is empty in JSON output, which is a bit awkward imo. I'd rather see Requested and Resolved as the same version.

You could probably use MarshalStructured and MarshalOutput on the `resolvedVersion` struct to achieve this.

Reply:
It doesn't appear that adding `MarshalStructured` to a struct field that itself has a `MarshallStructured` method will result in the first function being called. I've had to make that explicit, but it still works.